packages/types/src/utils.ts

```diff
@@ -24,6 +24,7 @@ export type TuplifyUnion<T, L = LastOf<T>, N = [T] extends [never] ? true : fals
   : Push<TuplifyUnion<Exclude<T, L>>, L>;
 
 export type Unarray<T> = T extends Array<infer U> ? U : T;
+export type NullableArray<T> = T extends Array<infer U> ? (U | null | undefined)[] : T[];
```

Comment:
Doing `NullableArray<PluginType>` will result in `PluginType[]` without `null`, is that intended?

Reply:
Yes this is just to `null` values in an array so if it just a primitive it shouldn’t do anything. So it should just return `T`. I think better is `Nullable<T>` will fix it

Reply:
What do you think if you don’t pass  an array should it return it back as is Ike null the values?

Reply:
It can just no: `type NullableArray<T> = Array<T | null>` ?